pkg/cmd/eventlistener/describe.go

```diff
@@ -179,6 +184,22 @@ func describeEventListenerOutput(w io.Writer, p cli.Params, f *cliopts.PrintFlag
 	return printer.PrintObject(w, el, f)
 }
 
+func describeEventListenerOutputURL(w io.Writer, p cli.Params, name string) error {
+	cs, err := p.Clients()
+	if err != nil {
+		return err
+	}
+
+	el, err := cs.Triggers.TriggersV1alpha1().EventListeners(p.Namespace()).Get(name, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(w, "%s\n", el.Status.Address.URL.String())
```

Comment:
Can we use this function here https://github.com/tektoncd/cli/blob/master/pkg/cmd/eventlistener/describe.go#L226